build/build_queryservice.sh

```diff
@@ -2,15 +2,21 @@
 set -e
 
 TARBALL="service-$QUERYSERVICE_VERSION-dist.tar.gz"
-TEMP_DIR="$(mktemp -d)"
+TARBALL_SIGNATURE="$TARBALL".md5
 
-wget "https://archiva.wikimedia.org/repository/releases/org/wikidata/query/rdf/service/$QUERYSERVICE_VERSION/$TARBALL" \
-    -O "$TEMP_DIR/$TARBALL"
+MD5_SIGNATURE="$(wget -qO- https://archiva.wikimedia.org/repository/releases/org/wikidata/query/rdf/service/$QUERYSERVICE_VERSION/$TARBALL_SIGNATURE)"
+CACHE_PATH="git_cache/queryservice/$MD5_SIGNATURE"
```

Comment:
maybe this should be in the cache folder rather than the git_cache folder as it's not really a git cache. thoughts?

Reply:
Probably… the query service release is not a git repo AFAIS.

Reply:
moving it there